docker-compose.yml

```diff
@@ -35,7 +36,7 @@ services:
       - 'rabbitmq_data:/data'
       - ./.docker/rabbitmq/etc/:/etc/rabbitmq/
       - ./.docker/rabbitmq/data/:/var/lib/rabbitmq/
-      - ./.docker/rabbitmq/logs/:/var/log/rabbitmq/
+      # - ./.docker/rabbitmq/logs/:/var/log/rabbitmq/
```

Comment:
@ketanreliefapps is that needed to remove ?

Reply:
no, it needed remove for my local